### PR TITLE
Fix build procces

### PR DESCRIPTION
**Pull request by OmidGhotbi**
fix the issue of setup.bat that not work on windows and make it more dynamic

### CODE REVIEW

**Review by u3dreal — COMMENTED** [review]
I fixed it

File: scripts/build_molecular.py
[the remark this answers is not in the document]
Comment (OmidGhotbi) [reply scripts/build_molecular.py:1]: i did not touch the mac os build, I can check it.

[the remark this answers is not in the document]
Comment (u3dreal) [reply scripts/build_molecular.py:1]: the build_molecular is also used by macos and you did not make case for macos so there would have been no chdir in the build process. but never mind it should work now

[the remark this answers is not in the document]
Comment (OmidGhotbi) [reply scripts/build_molecular.py:1]: sure i'll fix it today

**Pull request merged**